src/main/scala/ai/starlake/extract/JdbcDbUtils.scala

```diff
@@ -1278,11 +1278,12 @@ object JdbcDbUtils extends LazyLogging {
     auditColumns: Columns
   )(implicit settings: Settings): Try[Unit] = {
     val dataColumnsProjection = tableExtractDataConfig.columnsProjectionQuery(extractConfig.data)
+    val where = extractConfig.jdbcSchema.where.map(w => s"where $w").getOrElse("")
```

Comment:
We should rename it to extraCondition or rename above one to where in order to have the same name for both function

Reply:
Done